tests/yam/agama/import_agama_profile.pm

```diff
@@ -7,22 +7,26 @@
 use base Yam::Agama::patch_agama_base;
 use strict;
 use warnings;
-use testapi qw(assert_script_run data_url get_required_var select_console script_run);
+use testapi qw(assert_script_run data_url get_required_var set_var get_var select_console script_run);
 use autoyast qw(expand_agama_profile generate_json_profile);
 
 sub run {
     my $profile = get_required_var('AGAMA_PROFILE');
     my $profile_url = ($profile =~ /\.libsonnet/) ?
       generate_json_profile($profile) :
       expand_agama_profile($profile);
+    set_var('AGAMA_PROFILE', $profile_url);
```

Comment:
I don't get this, you have the variable, why you set it with a different value, a bit obscure, I guess just use the variable is enough.